app/src/main/java/com/example/solidarity/ProfileFragment.java

```diff
@@ -68,9 +75,8 @@ public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceStat
         FrameLayout flayout = view.findViewById(R.id.fLayout);
         profImage = view.findViewById(R.id.ivProfileImage);
         swipeContainer = (SwipeRefreshLayout) view.findViewById(R.id.swipeContainer);
-        /**profImage = new CircleImageView(getContext());
```

Comment:
no dead code

Reply:
I had just removed this unused code in this PR- all irrelevant comments should be gone